code/mapping/tests/mapping_common/test_general.py

```
@@ -0,0 +1,13 @@
+from mapping_common.map import Map
+
+import test_entity
+
+
+def test_map_conversion():
+    m = Map()
+    for i in range(200):
```

Ruff (0.8.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)